examples/reference/elements/matplotlib/Bounds.ipynb

```diff
@@ -36,9 +36,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Bounds (color='orange' linewidth=6)\n",
     "penguins = hv.RGB.load_image('../assets/penguins.png')\n",
-    "penguins * hv.Bounds((-0.15, -0.4, 0.2, 0))"
+    "penguins * hv.Bounds((-0.15, -0.4, 0.2, 0)).opts(color='orange', linewidth=6)"
```

Comment:
Should refactor this, here's what I would do:

```suggestion
    "bounds = hv.Bounds((-0.15, -0.4, 0.2, 0))\n",
    "penguins * bounds.opts(color='orange', linewidth=6)"
```

The current guide would suggest:

```python
(penguins * bounds).opts(
    opts.Bounds(color='orange', linewidth=6)
```

Personally my preference would be to declare elements first, and then do composition and styling in one step but I'd be okay going this route.

Reply:
I prefer the latter suggestion. I really don't like mixing `.opts` into a composition - it should only ever come at the end imho.